.github/workflows/wheel_win_x86_64.yml

```
@@ -151,7 +151,8 @@ jobs:
           CIBW_TEST_REQUIRES: numpy~=1.21 scipy pytest pytest-cov pytest-mock flaky
 
           CIBW_BEFORE_TEST: |
-            pip install git+https://github.com/PennyLaneAI/pennylane.git@master
+            pip install git+https://github.com/PennyLaneAI/pennylane.git@make-zstd-optional
+            #pip install git+https://github.com/PennyLaneAI/pennylane.git@master
```

```suggestion
            pip install git+https://github.com/PennyLaneAI/pennylane.git@master
```
Will be changed when https://github.com/PennyLaneAI/pennylane/pull/3257 is merged

---

Hey @chaeyeunpark, they just merged the PR. 
You can commit your own suggestion, since this will at the same time trigger the CI.